site/index.html

```diff
@@ -2,10 +2,119 @@
+                    <li><a href="glossary.html">Glossary</a></li>
+
+                </ul>
+            </nav>
+
+        </header>
+        <main>
+            <h1>Welcome to Saad's Website!</h1>
+            <p>My Name is Saad. I am from Ahmedabad,India. I am currently studying Computer Programming at Sault College.</p>
+            <p>I am Gym freak person. I love interacting with people and to make new friends and hang out with them. Apart from this I like Cycling, Swimming, Reading and playing Cricket.</p>
+            <h2>My Favourite Things</h2>
+            <a href="https://www.fcbarcelona.com/en/">
+            <img src="images/logo.png" alt="Favorite Logo" style="width:200px;"/>
+            </a>
+            <h3>Favourite Games</h3>
+            <ol>
+                <li>Call Of Duty</li>
+                <li>Subway City</li>
+                <li>GTA Vice City</li>
+            </ol>
+
+            <h3>Haiku</h3>
+            <p>
+                JANUARY<br>
+                Delightful display<br>
+                Snowdrops bow their pure white heads<br>
+                To the sun's glory.<br><br>
+
+                FEBRUARY<br>
+                Fresh green buds appear<br>
+                Indicating spring will soon<br>
+                Energize us all.<br><br>
+
+                MARCH<br>
+                Lambs gambol in fields<br>
+                Frisky with the joys of life<br>
+                Bleating happily.<br><br>
+                
+                APRIL<br>
+                Bluebells stand so proud<br>
+                Beneath trees so sparsely dressed<br>
+                Fresh green leaves unfold.<br><br>
+
+                MAY<br>
+                Much awaited sound<br>
+                Echoes heard amid dense trees<br>
+                Cuckoo has arrived.<br><br>
+
+                JUNE<br>
+                Parks and gardens burst<br>
+                With sounds and vibrant colours<br>
+                Perfect harmony.<br><br>
+
+                JULY<br>
+                Beaches become full<br>
+                Of families having fun<br>
+                In sand and big waves.<br><br>
+
+                AUGUST<br>
+                Ripe golden harvest<br>
+                Burning sun in azure skies<br>
+                Labours rewarded.<br><br>
+
+                SEPTEMBER<br>
+                Swallows congregate<br>
+                On telephone wires ready<br>
+                To migrate down south.<br><br>
+
+                OCTOBER<br>
+                Red and gold leaves fall,<br>
+                Crunchy as cornflakes beneath<br>
+                Feet on a crisp morn.<br><br>
+
+                NOVEMBER<br>
+                Frosty webs sparkle<br>
+                In the early morning sun<br>
+                Brightly bejeweled.<br><br>
+
+                DECEMBER<br>
+                First few flakes of snow<br>
+                Dust gardens like icing on<br>
+                A chocolate cake.<br><br>
```

Comment:
Choosing just one would be fine:
```suggestion
                First few flakes of snow<br>
                Dust gardens like icing on<br>
                A chocolate cake.<br>
```

Reply:
Totally optional